staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -137,6 +138,21 @@ func TestSyncLoadBalancerIfNeeded(t *testing.T) {
 			expectOp:          deleteLoadBalancer,
 			expectPatchStatus: false,
 		},
+		{
+			desc: "service specifies custom LB",
+			service: &v1.Service{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "with-external-balancer",
+					Namespace: "default",
+				},
+				Spec: v1.ServiceSpec{
+					Type:                  v1.ServiceTypeLoadBalancer,
+					LoadBalancerClassName: utilpointer.StringPtr("custom-loadbalancer"),
+				},
+			},
+			expectOp:          deleteLoadBalancer,
```

Comment:
`deleteLoadBalancer` is just the default op right?

Reply:
Can we set the other test-case fields explictily instead of using zero values? Easier to read this test table that way